lib/howl/ui/editor.moon

```diff
@@ -279,11 +281,10 @@ class Editor extends PropertyObject
     }
 
   transform_active_lines: (f) =>
-    lines = @active_lines
```

Comment:
This actually looks like an earlier oversight - it would be better to use this local var instead of refetching the active lines all the time.